plugins/infinispan100/src/main/java/org/radargun/service/Infinispan100ServerService.java

```diff
@@ -0,0 +1,162 @@
+package org.radargun.service;
+
+import java.io.IOException;
+import java.nio.file.FileSystems;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.ScheduledThreadPoolExecutor;
+import java.util.regex.Pattern;
+import java.util.stream.Collectors;
+
+import org.radargun.Service;
+import org.radargun.config.Property;
+import org.radargun.traits.Clustered;
+import org.radargun.traits.ProvidesTrait;
+import org.radargun.utils.Utils;
+
+import static org.radargun.service.ServerStartPattern.*;
+
+@Service(doc = InfinispanServerService.SERVICE_DESCRIPTION)
+public class Infinispan100ServerService extends Infinispan80ServerService {
+
+
+   @Property(doc = "Default server port for Hotrod, Rest and other operations. If this property is set a non-WildFly based server will be used.")
+   private String defaultServerPort;
+
+//   @Property(doc="")
+//   private String restServer;
+
+   @Property(doc="jgruoups config. If set, it will replace the default <jgroups> config")
+   private String jgroupsConfig;
+   @Property(doc="cache-container config. If set, it will replace the default <cache-container> config")
+   private String cacheContainerConfig;
+   @Property(doc="interfaces config. If set, it will replace the default <interfaces> config")
+   private String interfacesConfig;
+   @Property(doc="socket bindins config. If set, it will replace the default <socket-bindings> config")
+   private String socketBindingsConfig;
+   @Property(doc="endpoints config. If set, it will replace the default <endpoints> config")
+   private String endpointsConfig;
+   @Property(doc="security config. If set, it will replace the default <security> config")
+   private String securityConfig;
+
+   protected Clustered clustered;
+
+   @Override
+   public void init() {
+      if (isEapServer()) {
+         //call wildfly server environment
+         super.init();
+         return;
+      }
+
+      executor = new ScheduledThreadPoolExecutor(executorPoolSize);
+      clustered = new Infinispan100ServerClustered(this, defaultServerPort);
+      lifecycle = new InfinispanServerLifecycle(this) {
+
+         @Override
+         protected Pattern getStartOK() {
+            if(isEapServer()) {
```

Comment:
@alanfx IMHO, we don't need to have that. ISPN10 is about server-ng.

Reply:
Hey @diegolovison, I guess I am not opposed to leaving this here, since ISPN10 will still include the layered server even though it is deprecated